Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestCaseManager.java

```diff
@@ -49,21 +59,81 @@ synchronized static AutoIngestCaseManager getInstance() {
      * Constructs an object that handles locating and opening cases created by
      * auto ingest.
      */
-    private AutoIngestCaseManager() {
+    private AutoIngestCaseManager() throws AutoIngestCaseManagerException {
+        try {
+            init();
+        } catch (AutoIngestCaseManager.AutoIngestCaseManagerException ex) {
+            Logger.getLogger(AutoIngestCaseManager.class.getName()).log(Level.SEVERE, null, ex);
+            throw ex;
+        }
+    }
+    
+    /**
+     * Initialize the coordination service.
+     * 
+     * @throws org.sleuthkit.autopsy.experimental.autoingest.AutoIngestCaseManager.AutoIngestCaseManagerException 
+     */
+    private void init() throws AutoIngestCaseManager.AutoIngestCaseManagerException {
+        try {
+            coordinationService = CoordinationService.getInstance();
+        } catch (CoordinationServiceException ex) {
+            throw new AutoIngestCaseManager.AutoIngestCaseManagerException(ex.getMessage(), ex);
+        }
     }
 
-    /*
+    /**
      * Gets a list of the cases in the top level case folder used by auto
      * ingest.
      */
+    @Messages({
+        "AutoIngestCaseManager.CasePathsError=An error occurred while trying to retrieve the list of case paths."
+    })
     List<AutoIngestCase> getCases() {
         List<AutoIngestCase> cases = new ArrayList<>();
-        List<Path> caseFolders = PathUtils.findCaseFolders(Paths.get(AutoIngestUserPreferences.getAutoModeResultsFolder()));
-        for (Path caseFolderPath : caseFolders) {
-            cases.add(new AutoIngestCase(caseFolderPath));
+        List<Path> casePathList;
+        try {
+            casePathList = getCasePaths();
+            for (Path casePath : casePathList) {
+                cases.add(new AutoIngestCase(casePath));
+            }
+        } catch (AutoIngestCaseManagerException ex) {
+            String errorMessage = NbBundle.getMessage(AutoIngestCaseManager.class, "AutoIngestCaseManager.CasePathsError");
```

Comment:
I don't see any AutoIngestCaseManager messages in the Bundle.properties file. However, per our coding standard, we don't internationalize/localize exception messages, so just use some suitable English text here, given that we will not be displaying the message for the user (see next comment), thanks!